quickstart/101-machine-learning/readme.md

```diff
@@ -0,0 +1,33 @@
+# Azure Machine Learning workspace
+
+This deployment template specifies an [Azure Machine Learning workspace](https://docs.microsoft.com/en-us/azure/machine-learning/concept-workspace), 
```

Comment:
This is a bit of a nit, but in Terraform these are "configurations", not "templates".

Reply:
Much better name actually, will update it and as well in our docs